### DISCUSSION TITLE
Emote window in tab split

### DISCUSSION THREAD

**Discussion by Sacred-Shadow** [Chatterino/chatterino2#5003] [Ideas]
Unfortunately, the emote window only refers to the tab in which you have opened it.
A cool feature would be that I could also insert the emote window into the chat for a shared tab.

![image](https://github.com/Chatterino/chatterino2/assets/61997661/e7902762-abf9-43f4-b426-552247514b76)

Thus, if one wishes, they could have the emote menu open for each chat individually.

You could make it look like this.
![image](https://github.com/Chatterino/chatterino2/assets/61997661/7ffbfac5-6653-4510-97aa-ab4d38e7b751)

**Comment by 8thony** [comment]
Related Issues:
- #2430
- #2280
